package.json

```diff
@@ -28,10 +28,8 @@
     "contracts:compile:abi": "typechain --target ethers-v5 --out-dir src/abis/types \"./src/abis/**/*.json\"",
     "contracts:compile:v3": "typechain --target ethers-v5 --out-dir src/types/v3 \"./node_modules/@uniswap/**/artifacts/contracts/**/*.json\"",
     "contracts:compile": "yarn contracts:compile:abi && yarn contracts:compile:v3",
-    "prei18n:extract": "touch src/locales/en-US.po",
```

Comment:
are you sure we don't need this step anymore?

Reply:
yes - this file is created by `lingui extract --locale en-US`